data/coalition/coalition outfits.txt

```diff
@@ -249,6 +249,29 @@ outfit "Large Steering Module"
 	"turning heat" 8.2
 	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
 
+outfit "Refuelling Module"
+	category "Systems"
+	cost 822000
+	thumbnail "outfit/refuelling module"
+	"mass" 9
+	"outfit space" -9
+	"fuel capacity" -100
+	"fuel generation" .05
+	"energy consumption" .025
+	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, Resistance ships employ this outfit when carrying supplies or stolen outfits."
```

Comment:
```suggestion
	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, the Resistance have created an outfit capable of refuelling a ship in the middle of flight for ships carrying contraband."
```

Reply:
didn't do exactly those changes